recipes/vtk/all/conanfile.py

```diff
@@ -0,0 +1,444 @@
+import os
+import re
+
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+class VTKConan(ConanFile):
+    name = "vtk"
+    # version = "9.0.1"  # DO NOT SUBMIT!!! Remove this line in final version of PR to conan-center-index
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://vtk.org/"
+    license = "BSD license"
```

Comment:
Please use SPDX identifier. There are quite some amount of BSD licenses, so please make sure to choose the right one

https://spdx.org/licenses/

Reply:
Done